source/install/install-nginx.rst

```diff
@@ -0,0 +1,55 @@
+.. _install-nginx:
+
+Installing NGINX Server
+=======================
+
+In a production setting, use a proxy server for greater security and performance of Mattermost.
+
+The main benefits of using a proxy are as follows:
+
+  -  SSL termination
+  -  HTTP to HTTPS redirect
+  -  Port mapping ``:80`` to ``:8065``
+  -  Standard request logs
+
+**To install NGINX on Ubuntu Server:**
+
+1. Log into the server that will host the proxy and open a terminal window.
+
+2. Install NGINX.
+
+  ``sudo apt-get install nginx``
+
+3. After the installation is complete, verify that NGINX is running.
+
+  ``curl http://localhost``
```

Comment:
Yes, it's intentional. By using localhost, you can copy/paste the line directly into a terminal and run it without the need to customize for the IP.